cypress/TestCases/FireboltCertification/FederatedData.feature

```diff
@@ -0,0 +1,115 @@
+Feature: FederatedData
+
+    @initialization
+    Scenario: Launch FCA for 'FederatedData'
+        Given the environment has been set up for 'FederatedData' tests
+        And 3rd party 'certification' app is launched
+        When '3rd party app' invokes the 'Firebolt' API to 'push entity details data'
+        And '3rd party app' invokes the 'Firebolt' API to 'push purchases'
```

Comment:
this should be discovery.purchaseContent

Reply:
updated